src/models/user.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema, Types } from "mongoose";
+import { IUserSchema } from "./user.types";
+
+const userSchema = new Schema<IUserSchema>({
+  _id: Types.ObjectId,
+  created: Date,
+  email: String,
+  username: String,
+  password: String,
```

Comment:
Obviously **not the actual password** but rather a hash... we should hash it on the clients device and only store/ send the hash to the server :lock: